lib/config/options/index.ts

```diff
@@ -68,6 +68,31 @@ const options: RenovateOptions[] = [
     default: false,
     globalOnly: true,
   },
+  {
+    name: 'mergeConfidenceEndpoint',
+    description:
+      'If set, Renovate will query this API for Merge Confidence data.',
+    type: 'string',
+    advancedUse: true,
+    globalOnly: true,
+  },
```

Comment:
Can we set a default value here? I think it will make it easier for users to know how/what to proxy

Reply:
Yes we can. We fallback to `developer.mend.io` incase this value is `undefined`.